react/features/base/dialog/components/Dialog.native.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { translate } from '../../i18n';
+
+import Prompt from 'react-native-prompt';
```

Comment:
This import should be in the first group of imports for the 3rd party modules, right?